targets/targets.json

```diff
@@ -2836,6 +2836,111 @@
         ],
         "device_name": "STM32H747XIHx"
     },
+    "PORTENTA_H7": {
+        "core": "",
+        "public": false,
+        "inherits": ["MCU_STM32"],
```

Comment:
The commit was done before the unification of `MCU_STM32*` targets, so it can probably be rewritten in a "modern" way before merging. Thanks for pointing it out

Reply:
Done!